tests/exporter.js

```diff
@@ -65,29 +65,4 @@ define([
             );
         });
     });
-
-// TODO test with newlines (should they be preserved? old vellum did not)
-//    var TEST_XML_1 = util.xmlines('' +
```

Comment:
Removed xmlines. This is from at least almost 2 years ago, assuming if no one has asked to preserve newlines then they never will.